src/model/ProductReviewsModel.ts

```diff
@@ -38,14 +39,16 @@ async function insertReview(
 }
 
 async function getProductReviewByUser(
-  user_id: string,
+  user_id: number,
   product_id: string
 ): Promise<ProductReview | undefined> {
   const client = await getPostgresClient();
   let response = await client.query(
     `
     SELECT *
     FROM ProductReviews
+    INNER JOIN users ON productreviews.user_id = users.id
+    INNER JOIN products ON productreviews.product_id = products.id
```

Comment:
Are you sure this JOINs are doing anything?

Usually when you have a JOIN, you also have a clause from there in the `SELECT`. Otherwise what's the point of JOINing if you don't select anything from those tables?

For example

```
SELECT
  users.id,
  users.name,
  cars.id,
  cars.name
FROM users
INNER JOIN cars ON user.id = cars.owner_id
WHERE user.id = 12
```

Notice how in the above query we join with `cars` because we need something from there.

Reply:
having `SELECT *` selects everything and the `JOIN` actually gives all the necessary information